quetz/pkgstores.py

```diff
@@ -426,3 +454,15 @@ def get_filemetadata(self, channel: str, src: str):
         etag = infodata['etag']
 
         return (msize, mtime, etag)
+
+    def cleanup_temp_files(self, channel: str):
```

Comment:
this is a bit code duplication here. is there a way to make this a free function?

Reply:
we should tackle duplication in `S3` store and `Azure` store in a separate PR since this is a part of that now.